x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_bulk_actions/trial_license_complete_tier/perform_bulk_action_ess.ts

```diff
@@ -46,8 +47,15 @@ export default ({ getService }: FtrProviderContext): void => {
   const createWebHookConnector = () => createConnector(getWebHookAction());
 
   // Failing: See https://github.com/elastic/kibana/issues/173804
-  // Failing: See https://github.com/elastic/kibana/issues/182512
-  describe.skip('@ess perform_bulk_action - ESS specific logic', () => {
+  describe('@ess perform_bulk_action - ESS specific logic', () => {
+    before(async () => {
+      await esArchiver.load('x-pack/test/functional/es_archives/auditbeat/hosts');
```

Comment:
I wonder if we can tweak the test to expect any rule execution status except for 'error'? Loading the entire auditbeat archive seems like an overkill for that test scenario, given that the rule status is not relevant for it.

Reply:
Another option could be to use `*` as the rule index instead of `auditbeat-*`